Documentation/building/testing-with-corefx.md

```diff
@@ -111,11 +111,6 @@ After the tests are copied to the remote machine, you want to run them. Use one
 [tests\scripts\run-corefx-tests.sh](https://github.com/dotnet/coreclr/blob/master/tests/scripts/run-corefx-tests.sh)
 to run all the tests (consult the scripts for proper usage). Or, run a single test as described below.
 
-## Other corefx build considerations
```

Comment:
Updated instructions as since we're updating everything to match, corefx should now be building with clang9.0